views/css/other-landing.css

```diff
@@ -36,6 +36,15 @@ form input.form-control{
   position: fixed;
   width: 255px;
 }
+.project-img{
+  width: 15px;
+  height: 15px;
+  display: inline-block;
+  margin-right: 5px;
+
+  background-image: url('/img/eventcd (2).jpg');

```

Comment:
it'll be good if you can rename the image to something meaningful

Reply:
@devshiva619 are there any services for it or we need to hardcode the image as of now?

Reply:
@invincibleJai as of now its hardcoded, the values will be populated from the backend.